libvast/src/system/importer.cpp

```diff
@@ -226,9 +232,18 @@ void importer_state::send_report() {
   using namespace std::string_literals;
   auto elapsed = std::chrono::duration_cast<duration>(now - last_report);
   auto node_throughput = measurement{elapsed, measurement_.events};
+  auto num_schemas_seen = schema_counters.size();
+  std::vector<performance_sample> samples = {};
+  samples.reserve(num_schemas_seen + 2);
+  samples.push_back(performance_sample{"importer"s, measurement_});
+  samples.push_back(performance_sample{"node_throughput"s, node_throughput});
+  for (const auto& [name, count] : schema_counters)
+    samples.push_back(performance_sample{
+      "ingest", measurement{elapsed, count}, {{"schema", name}}});
```

Comment:
One additional note: Before merging, please add a changelog entry and update the docs section that lists all metrics.